### DISCUSSION TITLE
Newletter sign up form not working

### DISCUSSION THREAD

**Discussion by flonUK** [swellstores/community#485] [Help] [answered]
The sign-up form for the newsletter in the Horizon theme's footer does not submit or store customer emails.

Thanks

**Comment by logeshswell** [comment, marked as answer]
Yes, it's a gap in our implementation. You can add support for it by updating the customer field (email_optin) in GraphQL.
https://developers.swell.is/frontend-api/accounts#update-the-account-metadata
https://developers.swell.is/backend-api/accounts/the-account-model
